src/hooks/screen/useChatScreenHook.ts

```diff
@@ -208,29 +227,21 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           );
 
           await currentChatSchema?.save(localDb);
-          initChatData();
+          handleChatData();
         } else {
-          queue.addPriorityJob({
-            operationLabel: DatabaseOperationLabel.ChatScreen_SendChat,
-            id: `${selectedChannel?.id}-${new Date().valueOf()}`,
-            priority: QueueJobPriority.HIGH,
-            forceAddToQueue: true,
-            task: async () => {
-              currentChatSchema = await ChatSchema.generateSendingChat(
-                randomId,
-                userId,
-                selectedChannel?.id || '',
-                message,
-                attachments,
-                localDb,
-                'regular',
-                'pending'
-              );
+          currentChatSchema = await ChatSchema.generateSendingChat(
+            randomId,
+            userId,
+            selectedChannel?.id || '',
+            message,
+            attachments,
+            localDb,
+            'regular',
+            'pending'
+          );
 
-              await currentChatSchema?.save(localDb);
-              initChatData();
-            }
-          });
+          await currentChatSchema?.save(localDb);
+          handleChatData();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `useDatabaseQueueHook` has been replaced with direct database operations and the newly introduced `handleChatData` function. This simplifies the code and makes it easier to understand. However, it's important to ensure that the removal of `useDatabaseQueueHook` does not affect the application's performance or functionality.
```diff
-          initChatData();
+          handleChatData();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->